site/src/components/banner.md

```diff
@@ -33,3 +33,23 @@ Ensure the user knows they need to wait for some event in order to interact with
 	<p>Loading banner.</p>
 </div>
 ```
+
+### Dismissing banner
+
+Banner can be dismissed in conjunction usage with `dismiss` component. Read more about it [here](./dismiss.md).
+
+```html
+<div class="banner" id="example-banner-id" data-dismissable disappearing="slide-up">
```

Comment:
See comment above - I don't think we should use `disappearing` as an attribute name. This should be made into a data attribute.

Reply:
Updated